Updates peer dependencies

LaszloWa opened the pull request:
Updates peer dependencies to work with projects using React 17.x

asbjornh commented:
`react` and `react-dom` also need to be updated in `devDependencies` :)

asbjornh commented:
Why is there a diff on `yarn.lock`? I don't think changing `peerDependencies` should result in changes to that file

LaszloWa committed:
Upgrades react version in dev and peer depencies

LaszloWa commented:
> Why is there a diff on `yarn.lock`? I don't think changing `peerDependencies` should result in changes to that file

I am not sure. Removed it from the commit now.

> `react` and `react-dom` also need to be updated in `devDependencies` :)

Ok, added it!
